controllers/userController.js

```diff
@@ -147,4 +148,45 @@ const deleteUserAddresses = async (req, res) => {
     res.status(500).send({ message: err })
   }
 }
-module.exports = { userRegister, login, getUserData, deleteUserData, limitUsersData, userAddress, deleteUserAddresses }
+
+const forgotPassword = async (req, res) => {
+  const { username } = req.body
+  try {
+    const userData = await userModel.findOne({ where: { username } })
+    if (userData) {
+      const passwordResetToken = jwt.sign({
+        data: `${userData.username} ${userData.password}`
+      }, process.env.jwtSecKey);
+      await userModel.update({ passwordResetToken: passwordResetToken }, { where: { username } })
+      res.status(200).send(passwordResetToken)
+
+    } else {
+      res.status(404).send({ message: "user not found, please check the username" })
+    }
+
+  } catch (err) { res.status(500).send(err) }
+
+}
+const verifyResetPasswordToken = async (req, res) => {
+  const resetToken = req.params.passwordResetToken
+  const { new_password } = req.body
+  try {
+    const verifyToken = jwt.verify(resetToken, process.env.jwtSecKey, { expiresIn: 60 * 10 })
+    if (new_password) {
```

Comment:
new_password this check should be on the top, before doing any thing. @ashok-choudhary-0